src/main.ts

```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
   for (const task of executionGraphResult["actions"]) {
     if (task["tests"]) {
       core.info(
-        `${ansi.bold(task["action_id"])}: ${ansi.bold(
+        `${ansi.bold(task["action_id"])} ${ansi.bold(" action:")} ${ansi.bold(
+          task["action_id"]["status"]
+        )} » ${ansi.bold("Tests:")} ${ansi.bold(
           ansi.green(task["tests"]["passed"])
         )} ${ansi.bold(ansi.green(" passed"))}, ${ansi.bold(
           ansi.yellow(task["tests"]["skipped"])
         )} ${ansi.bold(ansi.yellow(" skipped"))}, ${ansi.bold(
           ansi.red(task["tests"]["failed"])
         )} ${ansi.bold(ansi.red(" failed"))}`
       )
-    } else if (task["passed"] === true) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.green("passed")}`))
-    } else if (task["passed"] === false) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.red("failed")}`))
-    }
-    if (task["vulnerabilities"]) {
+    } else if (task["passed"] === true && task["vulnerabilities"]) {
```

Comment:
Why `vulnerabilities` is a task? What we have is the action `trivy`

Reply:
this was already like it is before last changes. If tasks have vulnerabilities, they are shown that way.

Reply:
Got it.